README.md

```diff
@@ -5,7 +5,7 @@ subsequent levels of decentralization. Please refer to the project's
 [documentation pages](https://blockchain-technology-lab.github.io/consensus-decentralization/) for details on its architecture,
 required input, produced output, and more.
 
-Currently, the supported blockchains are:
+Currently, the supported blockchains are :
```

Comment:
I don't think it's common to have a space before a colon, so could you please remove the added spaces?